vm_setup_vars.yml

```diff
@@ -15,12 +15,14 @@ flavors:
     disk: "{{ lookup('env', 'MASTER_DISK') | default('30', true) }}"
     vcpu: "{{ lookup('env', 'MASTER_VCPU') | default('8', true) }}"
     extradisks: "{{ lookup('env', 'VM_EXTRADISKS') | default(false) }}"
+    extradisks_list: "{{ lookup('env', 'VM_EXTRADISKS_LIST').split(' ') | default(['vdb']) }}"
 
   worker:
     memory: "{{ lookup('env', 'WORKER_MEMORY') | default('8192', true) }}"
     disk: "{{ lookup('env', 'WORKER_DISK') | default('20', true) }}"
     vcpu: "{{ lookup('env', 'WORKER_VCPU') | default('4', true) }}"
     extradisks: "{{ lookup('env', 'VM_EXTRADISKS') | default(false) }}"
+    extradisks_list: "{{ lookup('env', 'VM_EXTRADISKS_LIST').split(' ') | default(['vdb']) }}"
```

Comment:
I think this needs to be a variable defined outside of the flavors block, see: 

https://github.com/metal3-io/metal3-dev-env/blob/master/vm-setup/roles/libvirt/defaults/main.yml#L24

vs extradisks which is defined per-flavor:

https://github.com/metal3-io/metal3-dev-env/blob/master/vm-setup/roles/common/defaults/main.yml#L22

Reply:
Thanks for looking at it. This did fix the issue when I got around to testing it